spec/services/device_tracking/manage_device_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe DeviceTracking::ManageDevice do
+  subject { described_class }
+
+  let(:user) { create(:user, :signed_up) }
+
+  context 'user signs on from an existing device' do
```

Comment:
Do we also need a context for 'user signs in from a new device'?

Reply:
Can add. That code is already exercised by new_device_tracking_spec.